common/app/views/fragments/standfirst.scala.html

```diff
@@ -16,7 +16,7 @@
     @content.standfirst.map { s =>
         @defining(Edition(request)) { edition =>
             @withJsoup(BulletCleaner(
-                if(content.isImmersive && content.isArticle) {
```

Comment:
I actually realised why it didn't work - because of the if statement it wasn't sure if it was getting a string or an option[string]  but anyway ship it as it is! :+1: